browser-test/src/error_pages.test.ts

```diff
@@ -1,24 +1,33 @@
-import {test} from '@playwright/test'
-import {
-  createTestContext,
-  gotoEndpoint,
-  NotFoundPage,
-  selectApplicantLanguage,
-} from './support'
+import {test, expect} from '@playwright/test' // Does not need to use civiform_fixture
+import {selectApplicantLanguage} from './support'
 
-test.describe('error pages', () => {
-  const ctx = createTestContext()
-  test('404 page', async () => {
-    const {page} = ctx
+test.describe(
+  'Error pages',
+  {tag: ['@uses-fixtures', '@parallel-candidate']},
```

Comment:
Just ignore `@parallel-candidate`. Trying to tag things that may potentially be able to be run in parallel some day. But don't count your chickens.